grass7/imagery/i.sentinel/i.sentinel.download/i.sentinel.download.py

```diff
@@ -174,6 +179,18 @@
     gs.fatal(_("Module requires pandas library: {}").format(e))
 
 
+def create_dir(dir):
+    if not os.path.isdir(dir):
+        try:
+            os.makedirs(dir)
+            return 0
+        except Exception as e:
+            gs.warning(_('Could not create directory {}').format(dir))
```

Comment:
I think black will complain here, we should use double quotes for messages

Reply:
Thanks, all single quotes in grass messages have now been changed to double quotes.